src/components/backend-ai-resource-policy-list.ts

```diff
@@ -440,15 +462,15 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
     render(
       html`
         <div id="controls" class="layout horizontal flex center"
-             .policy-name="${rowData.item.name}">
+            .policy-name="${rowData.item.name}">
         ${this.is_admin ? html`
               <wl-button fab flat inverted class="fg blue controls-running" icon="settings"
-                                 @click="${(e) => this._launchResourcePolicyDialog(e)}"><wl-icon>settings</wl-icon></wl-button>
-                                 ` : html``}
+                                @click="${(e) => this._launchResourcePolicyDialog(e)}"><wl-icon>settings</wl-icon></wl-button>
+                                ` : html``}
         ${this.is_admin ? html`
               <wl-button fab flat inverted class="fg red controls-running" icon="delete"
-                                 @click="${(e) => this._openDeleteResourcePolicyListDialog(e)}"><wl-icon>delete</wl-icon></wl-button>
-                                 ` : html``}
+                                @click="${(e) => this._openDeleteResourcePolicyListDialog(e)}"><wl-icon>delete</wl-icon></wl-button>
+                                ` : html``}
```

Comment:
Purpose of the one-space removal? I don't understand the criteria for this alignment change. (Although, I don't mean the original ones are correct.)

Reply:
Perhaps It's the same reason described above.